girder_worker/girder_plugin/web_client/JobStatus.js

```diff
@@ -1,45 +1,50 @@
-import JobStatus from '@girder/jobs/JobStatus';
+const events = girder.events;
 
-JobStatus.registerStatus({
-    WORKER_FETCHING_INPUT: {
-        value: 820,
-        text: 'Fetching input',
-        icon: 'icon-download',
-        color: '#89d2e2'
-    },
-    WORKER_CONVERTING_INPUT: {
-        value: 821,
-        text: 'Converting input',
-        icon: 'icon-shuffle',
-        color: '#92f5b5'
-    },
-    WORKER_CONVERTING_OUTPUT: {
-        value: 822,
-        text: 'Converting output',
-        icon: 'icon-shuffle',
-        color: '#92f5b5'
-    },
-    WORKER_PUSHING_OUTPUT: {
-        value: 823,
-        text: 'Pushing output',
-        icon: 'icon-upload',
-        color: '#89d2e2'
-    },
-    WORKER_CANCELING: {
-        value: 824,
-        text: 'Canceling',
-        icon: 'icon-spin3 animate-spin',
-        color: '#f89406'
-    }
-});
+// g:appload.before runs after all plugin static files have been loaded
+events.on('g:appload.before', () => {
```

Comment:
This is the new pattern for plugin code that depends on other plugin code and wants to mutate it at startup.